test/test_transforms_tensor.py

```diff
@@ -109,109 +109,118 @@ def _test_op(func, method, device, fn_kwargs=None, meth_kwargs=None, test_exact_
-    def test_color_jitter_brightness(self, brightness, device):
+    @pytest.mark.parametrize('channels', [1, 3])
+    def test_color_jitter_brightness(self, brightness, device, channels):
         tol = 1.0 + 1e-10
         meth_kwargs = {"brightness": brightness}
         _test_class_op(
             T.ColorJitter, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
-            tol=tol, agg_method="max"
+            tol=tol, agg_method="max", channels=channels,
         )
 
     @pytest.mark.parametrize('contrast', [0.2, 0.5, 1.0, 1.5, (0.3, 0.7), [0.4, 0.5]])
-    def test_color_jitter_contrast(self, contrast, device):
+    @pytest.mark.parametrize('channels', [1, 3])
+    def test_color_jitter_contrast(self, contrast, device, channels):
         tol = 1.0 + 1e-10
         meth_kwargs = {"contrast": contrast}
         _test_class_op(
             T.ColorJitter, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
-            tol=tol, agg_method="max"
+            tol=tol, agg_method="max", channels=channels
         )
 
     @pytest.mark.parametrize('saturation', [0.5, 0.75, 1.0, 1.25, (0.3, 0.7), [0.3, 0.4]])
-    def test_color_jitter_saturation(self, saturation, device):
+    @pytest.mark.parametrize('channels', [1, 3])
+    def test_color_jitter_saturation(self, saturation, device, channels):
         tol = 1.0 + 1e-10
         meth_kwargs = {"saturation": saturation}
         _test_class_op(
             T.ColorJitter, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
-            tol=tol, agg_method="max"
+            tol=tol, agg_method="max", channels=channels
         )
 
     @pytest.mark.parametrize('hue', [0.2, 0.5, (-0.2, 0.3), [-0.4, 0.5]])
-    def test_color_jitter_hue(self, hue, device):
+    @pytest.mark.parametrize('channels', [1, 3])
+    def test_color_jitter_hue(self, hue, device, channels):
         meth_kwargs = {"hue": hue}
         _test_class_op(
             T.ColorJitter, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
-            tol=16.1, agg_method="max"
+            tol=16.1, agg_method="max", channels=channels
         )
 
-    def test_color_jitter_all(self, device):
+    @pytest.mark.parametrize('channels', [1, 3])
+    def test_color_jitter_all(self, device, channels):
         # All 4 parameters together
         meth_kwargs = {"brightness": 0.2, "contrast": 0.2, "saturation": 0.2, "hue": 0.2}
         _test_class_op(
             T.ColorJitter, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
-            tol=12.1, agg_method="max"
+            tol=12.1, agg_method="max", channels=channels
         )
 
 
 @pytest.mark.parametrize('device', cpu_and_gpu())
 @pytest.mark.parametrize('m', ["constant", "edge", "reflect", "symmetric"])
 @pytest.mark.parametrize('mul', [1, -1])
-def test_pad(m, mul, device):
+@pytest.mark.parametrize('channels', [1, 3])
```

Comment:
I do not think that we need to add specific channel 1 test to geometric transforms like pad, crop etc

Reply:
Yeah, it's a little bit confusing to me at first because it's quite duplicated.

It's okay to remove and leave channels=3 as default, though.